es6-modules/src/components/todo-app.js

```diff
@@ -0,0 +1,55 @@
+import TodoList from './todo-list';
+import TodoForm from './todo-form';
+import { createElement, noop } from '../lib/index';
```

Comment:
When importing from an `index` file you don't need to include the file name. For example:
```JAVASCRIPT
import { createElement, noop } from '../lib';
```